app/utils/electronUtils.js

```diff
@@ -50,6 +50,10 @@ ${this.webContents.getURL()}`);
     }
   }
 
+  isOnExternalPage(): boolean {
+    return !this.webContents.getURL().substring(0,50).includes("immobilienscout24.de")
```

Comment:
I think this could be solved in a cleaner fashion using the built in `URL` object, but it doesn't matter.